portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintlabelview.js

```diff
@@ -82,7 +84,7 @@ var OncoprintLabelView = (function () {
 		    var previous_track_id = getLabelAboveMouseSpace(view, track_group, evt.offsetY, view.dragged_label_track_id);
 		    stopDragging(view, previous_track_id);
 		}
-		view.tooltip.hide();
+		view.tooltip.hideIfNotAlreadyGoingTo(150);
```

Comment:
I can't think of much right now. Renaming it to something like `initializeHideTimeout()` would only need *more* documentation to confirm that it wouldn't reset an already-existing timeout.